packages/storybook/stories/TextInput.stories.jsx

```diff
@@ -1,6 +1,6 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 
-import TextInput from '../../react-components/src/components/TextInput/TextInput';
+import { TextInput } from '@department-of-veterans-affairs/component-library';
```

Comment:
I haven't been able to pull your branch and test this, but I'm betting that if you go to the docs page for this story you'll see the prop table is empty. We probably don't want these changes committed.